Project.toml

```diff
@@ -0,0 +1,22 @@
+name = "Knockout"
+uuid = "bcebb21b-c2e3-54f8-a781-646b90f6d2cc"
+license = "MIT"
+version = "0.8.3"
```

Comment:
Current release is 0.2.2 so I imagine this should be 0.2.3?

Reply:
Yes! I got the minor from WebIO and the patch from Knockout... 🌮